### ISSUE TITLE
Missing Windows Binaries

### ISSUE THREAD

**Issue by glennsarti** [puppetlabs/wash#207]
**Describe the solution you'd like**
The README hints that wash works on windows, but there are no Windows builds

**Describe alternatives you've considered**
Building myself - No instructions and no prior art

**Additional context**
* Go is supposed to be cross platform
* WASH plugins says I can use any language I want, so I'd like to use powershell seem as though it's had a provider system since V1.0

**Comment by glennsarti** [comment]
Oh also, no mention of Windows on the Roadmap either.

**Comment by MikaelSmith** [comment]
That's true. Windows support minimally means getting the daemon working, and that needs alternatives to FUSE and UNIX sockets.

There's some sort of powershell provider that should be able to replace FUSE, and we can use named pipes instead of UNIX sockets. I'll add a section to the roadmap.

**Comment by MikaelSmith** [comment]
We also probably need to replace using extended attributes to map a path to an API call.

**Comment by MikaelSmith** [comment]
I have gotten Wash working under WSL 2. That doesn't mean native support isn't going to happen, but it is a means to try it out.
